src/node_worker.cc

```diff
@@ -132,7 +132,17 @@ class WorkerThreadData {
  public:
   explicit WorkerThreadData(Worker* w)
     : w_(w) {
-    CHECK_EQ(uv_loop_init(&loop_), 0);
+    if (int ret = uv_loop_init(&loop_) != 0) {
```

Comment:
`ret` is always 1 or 0, isn't it? That's not going to work with `uv_err_name_r()`.

Reply:
@bnoordhuis, when `ret` is defined first , the value is not 1. But when it is used in `if(condition)` the value is `1`. Why is that so ?

Reply:
you've written the equivalent of `int ret = (uv_loop_init(&loop_) != 0)`